doc/source/structures/misc/resource_transfer.rst

```diff
@@ -0,0 +1,100 @@
+.. _resource_transfer:
+
+Resource Transfer 
+=================
+
+Usage is covered elsewhere <../commands/resource_transfer.html>`__
```

Comment:
I'm fairly sure this link syntax is messed up but I'll get to it later when I'm on the unix boot where I have the stuff to parse the RST files installed.